flash/image/detection/model.py

```diff
@@ -126,6 +127,12 @@ def get_model(
         anchor_generator,
         **kwargs,
     ):
+        if _ICEVISION_AVAILABLE:
```

Comment:
Shouldn't need a check here as the backbones will only be registered if the library is available.